src/features/common/Layout/Footer/index.tsx

```diff
@@ -92,7 +96,13 @@ export default function Footer(): ReactElement | null {
 
   return ready ? (
     <footer>
-      <div className={styles.footerMainContainer}>
+      <div
+        className={styles.footerMainContainer}
+        style={{
+          backgroundColor:
+            pathname === '/sites/[slug]/t/[id]' ? '#fafaff' : '#fff',
+        }}
```

Comment:
This isn't needed. 

I'm assuming this is done to match the footer colour with the page background, but we could simply inherit the background colour, or leave the footer background transparent.